core/net/rpl/rpl.c

```diff
@@ -139,7 +139,7 @@ rpl_purge_routes(void)
       PRINTF("No more routes to ");
       PRINT6ADDR(&prefix);
       dag = default_instance->current_dag;
-      /* Propagate this information with a No-Path DAO to preferred parent if we are not a RPL Root */
+      /* Propagate this information with a No-Path DAO to preferred parent if we are not an RPL Root */
```

Comment:
It is impossible to know (that's why I thought I'd point it out) -- not sure if it even is in the RPL RFC :stuck_out_tongue_winking_eye: 

Good idea about adding it to the Wiki!